source/MRViewer/MRRenderDimensions.h

```diff
@@ -65,9 +78,16 @@ struct AngleParams
 
     // Whether we should draw a ray from the center point to better visualize the angle. Enable this if there isn't already a line object there.
     std::array<bool, 2> shouldVisualizeRay{ true, true };
+
+    // What object to select when the label is clicked.
+    // Optional. Not clickable if this is null.
+    const VisualObject* objectToSelect = nullptr;
+
+    // Optional. If specified, this name is drawn above the measurement.
+    std::string objectName;
```

Comment:
it seems to be common for several class, should we move it to separate struct? also, do we want to save tiny bit of memory using `const std::string*` instead of value?